src/runtime_src/tools/xclbin/SectionPartitionMetadata.cxx

```diff
@@ -364,6 +364,18 @@ SchemaTransformToDTC_addressable_endpoints( const boost::property_tree::ptree& _
   }
 }
 
+void
+SchemaTransformToDTC_partition_info ( const boost::property_tree::ptree& _ptOriginal,
+                                           boost::property_tree::ptree & _ptTransformed)
+{
+  std::ostringstream buf;
+  boost::property_tree::ptree ptInfo;
+
+  boost::property_tree::write_json(buf, _ptOriginal, false);
+  ptInfo.put("", buf.str());
+
+  _ptTransformed.add_child("__INFO", ptInfo);
```

Comment:
Hmm, "__INFO" looks out-of-place compared to the other schema variables....Is this really the case?

Reply:
added { "__INFO", FDTProperty::DF_sz }